packages/engine/src/schemas/scope/scope-type.schema.ts

```diff
@@ -35,7 +37,9 @@ export const scopeTypeMethods = ['find', 'get'] as const
 // Main data model schema
 export const scopeTypeSchema = Type.Object(
   {
-    type: Type.String(),
+    type: TypedString<ScopeType>({
+      format: 'uuid'
```

Comment:
@HexaField this shouldnt be uuid?

Reply:
Oh, perhaps. Does that affect the database?

Reply:
It doesn't but maybe at some time in future it will. I have made the fix in https://github.com/EtherealEngine/etherealengine/pull/9331